bindings/python/setup.py

```diff
@@ -250,7 +256,10 @@ def build_extension_with_b2(self):
             log.info(config.read())
             config.close()
             args.append(f"--project-config={config.name}")
-            self.spawn(["b2"] + args)
+
+            python_binding_dir = pathlib.Path(__file__).parent.absolute()
```

Comment:
I think you want `Path.resolve()`, not `Path.absolute()`. Not actually sure what `.absolute()` seems undocumented.

Reply:
actually, `resolve()` is problematic on windows. It doesn't only resolve symlinks, but also mapped drive letters, to their canonical UNC path. When running this in `cmd.exe` that fails because it doesn't support UNC paths.